Sources_v3/Workers/Background/WebSocketConnectedProcessor.swift

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+
+/// After WebSocket changes it's state to `connected` we need to start watching existing channels so we can receive updated on them.
+/// This background worker observes the WebSocket state and on `connected` we are fetching all channels from DB and sending
+/// ChannelListQuery to backend to start `watching` channels.
+final class WebSocketConnectedProcessor<ExtraData: ExtraDataTypes>: Worker {
```

Comment:
I think this is more about the channels than the web socket. My suggestion: `ChannelWatcher` :)